common/views/components/Buttons/ButtonControl/ButtonControl.config.js

```diff
@@ -0,0 +1,23 @@
+export const label = 'Control';
+export const status = 'graduated';
+export const collated = true;
+export const variants = [
+  {
+    name: 'default',
+    label: 'light',
+    context: {
+      icon: 'download',
+      extraClasses: 'button-control--light',
+      text: 'download'
+    }
+  },
+  {
+    name: 'default',
+    label: 'dark',
+    context: {
+      icon: 'download',
+      extraClasses: 'button-control--dark',
+      text: 'download'
+    }
+  }
+];
```

Comment:
Do we have reasons as to when we use dark and light?
If it just dependant on what colour they sitting on\?

Reply:
My understanding is it is completely dependent on what colour they sit on/around; decided on a case-by-case basis.